src/doc/guide.md

```diff
@@ -104,41 +103,61 @@ Once you’re ready for release, you can use `cargo build --release` to compile
 <span style="font-weight: bold"
 class="s1">   Compiling</span> hello_world v0.1.0 (file:///path/to/project/hello_world)</code></pre>
 
+`cargo build --release` puts the resulting binary in
+`target/release/hello_world` instead of `target/debug`.
```

Comment:
I would say "puts it in `target/release` instead of", it doesn't go in another folder named hello_world